WcaOnRails/app/webpacker/lib/show-schedule.js

```diff
@@ -10,6 +10,44 @@ window.wca.registerVenueData = (id, venueData) => {
   dataByVenueId[id] = venueData;
 };
 
+const HEX_BASE = 16;
+
+/**
+ * Convert a HEX color code to RGB values.
+ *
+ * @param {string} hexColor HEX color code to convert to RGB
+ *
+ * @returns {Array<number>} RBG values, defaults to `[0, 0, 0]` if `hexColor` cannot be parsed
+ */
+const hexToRgb = (hexColor) => {
+  if (/#[0-9A-Fa-f]{6}/.test(hexColor)) {
```

Comment:
I have no preference, but if you use the regex hexcolor.match(/.{1,2}/g) you can get all three parts of the string directly see https://convertingcolors.com/blog/article/convert_hex_to_rgb_with_javascript.html

Reply:
I'd like to keep the regex more specific. Would you be ok with `hexColor.match(/^#([0-9A-Fa-f]{2})([0-9A-Fa-f]{2})([0-9A-Fa-f]{2})$/)`, then I can use the capture groups?

Reply:
yeah sure